packages/botonic-core/src/index.d.ts

```diff
@@ -21,6 +21,10 @@ export const INPUT: {
   DOCUMENT: 'document'
   LOCATION: 'location'
   CONTACT: 'contact'
+  BUTTON_MESSAGE: 'buttonmessage'
+  CAROUSEL: 'carousel'
+  CUSTOM: 'custom'
+  WEBCHAT_SETTINGS: 'webchat_settings'
```

Comment:
why webchat_settings contains an underscore and buttonmessage doesn't?

Reply:
renamed to be consistent with current implementations